cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go

```diff
@@ -35,42 +35,169 @@ const (
+	86:  {description: "Novell OES", importerOSIDs: []string{}},
+	87:  {description: "Novell Linux Desktop", importerOSIDs: []string{}},
+	88:  {description: "Sun Java Desktop System", importerOSIDs: []string{}},
+	89:  {description: "Mandriva", importerOSIDs: []string{}},
+	90:  {description: "Mandriva 64-Bit", importerOSIDs: []string{}},
+	91:  {description: "TurboLinux", importerOSIDs: []string{}},
+	92:  {description: "TurboLinux 64-Bit", importerOSIDs: []string{}},
+	93:  {description: "Ubuntu", importerOSIDs: []string{}},
+	94:  {description: "Ubuntu 64-Bit", importerOSIDs: []string{"ubuntu-1404", "ubuntu-1604", "ubuntu-1804"}},
+	95:  {description: "Debian", importerOSIDs: []string{}},
+	96:  {description: "Debian 64-Bit", importerOSIDs: []string{"debian-8", "debian-9"}},
+	97:  {description: "Linux 2.4.x", importerOSIDs: []string{}},
+	98:  {description: "Linux 2.4.x 64-Bit", importerOSIDs: []string{}},
+	99:  {description: "Linux 2.6.x", importerOSIDs: []string{}},
+	100: {description: "Linux 2.6.x 64-Bit", importerOSIDs: []string{}},
+	101: {description: "Linux 64-Bit", importerOSIDs: []string{}},
+	102: {description: "Other 64-Bit", importerOSIDs: []string{}},
+	103: {description: "Microsoft Windows Server 2008 R2", importerOSIDs: []string{"windows-2008r2", "windows-2008r2-byol"}},
+	104: {description: "VMware ESXi", importerOSIDs: []string{}},
+	105: {description: "Microsoft Windows 7", importerOSIDs: []string{"windows-7-x64-byol", "windows-7-x86-byol"}},
+	106: {description: "CentOS 32-bit", importerOSIDs: []string{}},
+	107: {description: "CentOS 64-bit", importerOSIDs: []string{"centos-6", "centos-7", "centos-8"}},
+	108: {description: "Oracle Linux 32-bit", importerOSIDs: []string{}},
+	109: {description: "Oracle Linux 64-bit", importerOSIDs: []string{}},
+	110: {description: "eComStation 32-bitx", importerOSIDs: []string{}},
+	111: {description: "Microsoft Windows Server 2011", importerOSIDs: []string{}},
+	113: {description: "Microsoft Windows Server 2012", importerOSIDs: []string{"windows-2012", "windows-2012-byol"}},
+	114: {description: "Microsoft Windows 8", importerOSIDs: []string{"windows-8-x86-byol"}, nonDeterministic: true},
+	115: {description: "Microsoft Windows 8 64-bit", importerOSIDs: []string{"windows-8-x64-byol"}, nonDeterministic: true},
+	116: {description: "Microsoft Windows Server 2012 R2", importerOSIDs: []string{"windows-2012r2", "windows-2012r2-byol"}},
+	117: {description: "Microsoft Windows Server 2016", importerOSIDs: []string{"windows-2016", "windows-2016-byol"}},
+	118: {description: "Microsoft Windows 8.1", importerOSIDs: []string{}},
+	119: {description: "Microsoft Windows 8.1 64-bit", importerOSIDs: []string{}},
+	120: {description: "Microsoft Windows 10", importerOSIDs: []string{"windows-10-x86-byol"}, nonDeterministic: true},
+	121: {description: "Microsoft Windows 10 64-bit", importerOSIDs: []string{"windows-10-x64-byol"}, nonDeterministic: true},
+	//TODO: windows-2019, windows-2019-byol
```

Comment:
Is this TODO for a follow-up PR?

Reply:
yes, once windows-2019 becomes available in OVF spec. I've created a tracker bug.